torch/_inductor/__init__.py

```diff
@@ -30,6 +30,42 @@ def compile(
     return compile_fx(gm, example_inputs, config_patches=options)
 
 
+def aoti_compile_and_package(
+    package_path: str,
```

Comment:
To be more consistent with the current aot_compile API, should we make this as optional? That way we still generate the package in a tmp directory and all the aoti unit tests can continue to work without specifying a path when we migrate to this API.